release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -2833,3 +2833,29 @@ You can view the container images in this release by running the following comma
 $ oc adm release info 4.13.0 --pullspecs
 ----
 //replace 4.y.z for the correct values for the release. You do not need to update oc to run this command.
+[id="ocp-4-13-1"]
+=== RHSA-2023:3304 - {product-title} 4.13.1 bug fix and security update
+
+Issued: 2023-05-30
+
+{product-title} release 4.13.1, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:3304[RHSA-2023:3304] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:3303[RHSA-2023:3303] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.13.1 --pullspecs
+----
+
+[id="ocp-4-13-1-bug-fixes"]
+==== Bug fixes
+* Previously, assisted installations may have encounter a transient error, and if that occurred the installation failed to recover. With this update, transient errors are retried correctly. (link:https://issues.redhat.com/browse/OCPBUGS-13138[*OCPBUGS-13138*])
```

Comment:
```suggestion
* Previously, assisted installations could encounter a transient error. If that error occurred, the installation failed to recover. With this update, transient errors are re-tried correctly. (link:https://issues.redhat.com/browse/OCPBUGS-13138[*OCPBUGS-13138*])
```

TBH, not sure about this message. It's not clear to me if the chance of encountering transient errors is reduced or eliminated on re-tries.